packages/web-server/src/api/routers/catch-all.router.ts

```diff
@@ -1,10 +1,13 @@
 import express, { Request, Response } from 'express';
 import path from 'path';
+import { InjectEnvVariablesCommand } from '../../app';
 import { publicDirectory } from '../../common';
 
 // Default "catch-all" route serving the Ethereum Studio editor
 export const catchAllRouter = express.Router();
 
-catchAllRouter.get('/*', (_req: Request, res: Response) => {
+catchAllRouter.get('/*', async (_req: Request, res: Response) => {
+    const command = new InjectEnvVariablesCommand();
+    await command.execute();
     res.sendFile(path.join(publicDirectory, 'index.html'));
```

Comment:
Same here. Why not simply returning the processed file in the command, instead of writting to FS and then reading from it.

Reply:
Fixed